src/Query/Grammar.php

```diff
@@ -110,24 +119,23 @@ protected function compileIndexHint(Builder $query, $indexHint)
     }
 
     /**
-     * @param Builder $query
      * @param array<string, Expression|string> $where
      * @return string
      */
-    protected function whereInArray(Builder $query, $where)
+    protected function whereInArray(Builder $builder, $where)
```

Comment:
If you look at the code in base class, all similar method names the Builder argument as `$query`. 
This should be reverted.

Reply:
I made a mistake when I added `_` to the prefix of an unused parameter and then reverted it, as is my habit when using Psalm. I will change it back!